File: 01-business-sub-system/src/main/java/com/github/yingzhuo/datawarehouse/businesssubsys/dao/CommodityDao.scala
package com.github.yingzhuo.datawarehouse.businesssubsys.dao

import com.github.yingzhuo.datawarehouse.businesssubsys.domain.Commodity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
trait CommodityDao extends JpaRepository[Commodity, String] {
}

File: 01-business-sub-system/src/main/java/com/github/yingzhuo/datawarehouse/businesssubsys/dao/CommodityDao.java
package com.github.yingzhuo.datawarehouse.businesssubsys.dao;

import com.github.yingzhuo.datawarehouse.businesssubsys.domain.Commodity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface CommodityDao extends JpaRepository<Commodity, String> {
}